content/minutes/cab/2022-09-21-minutes.md

```diff
@@ -0,0 +1,41 @@
+---
+date: "2022-09-21T00:00:00+00:00"
+title: "September 21, 2022 Advisory Board Meeting"
+tags: ["Open Force Field Consortium", "Advisory Board", "minutes"]
+categories: ["Advisory Board Meeting Minutes"]
+draft: false
+description: "Meeting minutes"
+weight: 10
+author: "Diego Nolasco"
+---
+
+The [Open Force Field Consortium Advisory Board](https://openforcefield.org/about/organization/#open-force-field-consortium) met on September 21, 2022.
+The minutes are summarized as follows:
+
+#### Elections for the governing board (invitation to candidacy)
+
+* D. Nolasco announced the opening of a vacancy in the governing board. Additionally, he reminded everyone that it is common for us to hold elections to compose the governing board at this time of year. He committed to send the rules and regulations via email and to prepare a list of all candidates.
+
+#### OpenFF support renewal season
+
+* D. Nolasco announced that we have been thinking of strategies that would help our industrial partners when advocating on our behalf with their management. He presented a suggestion from J. Chodera considering the possibility that we could operate on the model known as freemium. We would still continue to make our software available free of charge to everyone, but with a year of delay when compared to what our industrial partners would have. That way, our industrial partners would receive the most up-to-date versions as soon as they were released while the rest of the community would have access to the previous version. The suggestion was not accepted and the members of the advisory board made an effort to propose possibilities that would allow us to continue producing open software and guaranteeing immediate access to the entire community.
+
+#### OpenFF "micro workshop series" - Genentech
+
+* B. Swope presented the results obtained at Genentech as a foundation for important feedback on Sage (OpenFF-2.0.0). In addition to pointing out positives and negatives, he suggested directions that could add value in a relevant way to his research. These would be:
+
+	- Charge model options
+	- Virtual sites
+	- Paramenters tuned for use with better water models
+	- (Limited) polarizability
+
+All his suggestions were taken into account and the OpenFF team will start a process of analyzing the possibilities with the interest of addressing as many demands and requests as possible.
```

Comment:
This sentence is weirdly worded ("demands" is a stronger term than really fits here). 

```suggestion
The OpenFF team gratefully welcomed his feedback and guidance, and looks forward to hearing from more industry partners in future meetings. 
```